src/index.js

```diff
@@ -169,24 +169,26 @@ const execCallback = function(req, res) {
   }
 
   if (
-      !args.threads &&
       args.override &&
       'options' in req.body &&
       't' in req.body.options &&
-      req.body.options.t
+      req.body.options.t &&
+      typeof req.body.options.t == "number"
   ) {
-    reqOptions.push('-t ' + req.body.options.t);
+    args.threads = req.body.options.t;
```

Comment:
This would override the default value at server level, so subsequent requests with no `t` option won't use the default setting any more. We should define a local `threads` variable initialized with `args.threads` before this check and update if required.